pkg/controllers/core/catalog_controller_test.go

```diff
@@ -316,6 +320,22 @@ var _ = Describe("Catalogd Controller Test", func() {
 					Expect(pack.Spec.Channels[0].Entries).To(HaveLen(1))
 					Expect(Expect(pack.Spec.Channels[0].Entries[0].Name).To(Equal(testBundleName)))
 				})
+
+				// TODO (rashmigottipati): Add testing of CatalogMetadata sync process.
+				It("should create CatalogMetadata resources", func() {
+					catalogMetadatas := &v1alpha1.CatalogMetadataList{}
+					Expect(cl.List(ctx, catalogMetadatas)).To(Succeed())
+					Expect(catalogMetadatas.Items).To(HaveLen(3))
+					for _, catalogMetadata := range catalogMetadatas.Items {
+						Expect(catalogMetadata.Name).To(ContainSubstring(testCatalogMetadataName))
```

Comment:
Instead of adding another variable, can we use `cKey.Name` here directly? (For clarity, perhaps we could change `cKey` to `catalogKey`)

Also, any reason not use use `Equal` instead of `ContainSubstring`?
```suggestion
						Expect(catalogMetadata.Name).To(Equal(catalogKey.Name))
```